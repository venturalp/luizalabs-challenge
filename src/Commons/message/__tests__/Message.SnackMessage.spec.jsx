import { render } from '@testing-library/react'
import { ThemeWrapper } from 'Commons/tests/Tests.Helpers'
import React from 'react'
import { SnackMessage } from '../Message.SnackMessage'

describe('test SnackMessage', () => {
  it('render snack message w default values', () => {
    const { asFragment } = render(
      <ThemeWrapper>
        <SnackMessage>foo bar</SnackMessage>
      </ThemeWrapper>,
    )

    expect(asFragment()).toMatchSnapshot()
  })
})
