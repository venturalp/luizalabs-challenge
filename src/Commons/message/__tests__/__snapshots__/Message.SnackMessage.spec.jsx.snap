// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test SnackMessage render snack message w default values 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ bEUqRb"
  >
    foo bar
  </div>
</DocumentFragment>
`;
